src/pages/docs/administration/private-cloud-migration/index.md

```diff
@@ -141,6 +142,10 @@ A complete migration involves:
     1. Pointing CI servers and external scripts to the cloud instance.
     1. Updating firewall rules to allow the cloud instance to connect to listening tentacles.
 
+:::div{.hint}
+EventExports is available from **2023.3** onwards as part of the audit log retention feature.
```

Comment:
I don't think this hint block is needed here

Reply:
Deleted